strawberry/utils/str_converters.py

```diff
@@ -1,11 +1,19 @@
+auto_camelcase = True
```

Comment:
Nitpick: We're being inconsistent about whether whether it's `camel_case` or `camelcase`

Reply:
Yeah I was debating that here. Graphene had the inconsistency so I was keeping it the same as that. I'll update it.